core/src/main/scala/filodb.core/memstore/PartKeyLuceneIndex.scala

```diff
@@ -498,14 +499,14 @@ class PartKeyLuceneIndex(ref: DatasetRef,
     .maximumSize(100)
     .recordStats()
     .build((key: (IndexReader, String)) => {
-      new DefaultSortedSetDocValuesReaderState(key._1, FACET_FIELD_PREFIX + key._2)
+      new DefaultSortedSetDocValuesReaderState(key._1, FACET_FIELD_PREFIX + key._2, new FacetsConfig())
```

Comment:
That third argument is now required, and the default-constructed `FacetsConfig` provides the same behavior as before [(source)](https://lucene.apache.org/core/9_7_0/facet/org/apache/lucene/facet/sortedset/DefaultSortedSetDocValuesReaderState.html#%3Cinit%3E(org.apache.lucene.index.IndexReader,java.lang.String)).